pkg/eventtype/eventtypes.go

```diff
@@ -44,14 +44,16 @@ type EventTypeAutoHandler struct {
 	Logger          *zap.Logger
 }
 
-// generateEventTypeName is a pseudo unique name for EvenType object based on on the input params
+// generateEventTypeName is a pseudo unique name for EvenType object based on the input params
 func generateEventTypeName(name, namespace, eventType, eventSource string) string {
-	suffixParts := eventType + eventSource + namespace + name
-	suffix := base64.StdEncoding.EncodeToString([]byte(suffixParts))[:10]
-	return utils.ToDNS1123Subdomain(fmt.Sprintf("%s-%s-%s", "et", name, suffix))
+	// Take last third of the event type as a seed
+	eventTypeSuffix := eventType[len(eventType)-len(eventType)/3:]
```

Comment:
Note: That's a leftover from my tests with base64 encoding. Probably doesn't have much added value now.

Reply:
@dsimansk do all of your new tests still pass without this? If so, I think we could probably just get rid of this

Reply:
Yep, they do. I'll remove it, just needed a second opinion here. :)